ChecksSelectionFunctionalityOfProductParameters_test.java

```diff
@@ -0,0 +1,54 @@
+package AutomationPracticeTestsSuite;
+
+import common.BaseTest;
+import org.assertj.core.api.SoftAssertions;
+import org.junit.Test;
+import pageobject.AutomationPracticeSite.AutomationPracticeMainPage;
+import pageobject.AutomationPracticeSite.ProductDetailsPage;
+import pageobject.AutomationPracticeSite.ShoppingCartPage;
+
+public class ChecksSelectionFunctionalityOfProductParameters_test extends BaseTest {
+
+    @Test
+    public void verifySelectionFunctionalityOfProductParameters() {
+
+        String navUrl = propertyHelper.readProperty("automation-practice.site.url");
+        webDriver.get(navUrl);
+
+        String expectedResultOfProductTitle;
```

Comment:
what is the purpose of declaring vars without immediate initialization?

Reply:
I can write differently. Declare a variable and assign a value at the same time.